modules/integration/src/test/scala/scala/cli/integration/SipScalaTests.scala

```diff
@@ -107,6 +107,19 @@ class SipScalaTests extends ScalaCliSuite {
       }
     }
 
+  if (TestUtil.isNativeCli)
+    test(s"usage instruction should point to scala when installing by cs") { // https://github.com/VirtusLab/scala-cli/issues/1662
+      TestInputs.empty.fromRoot {
+        root => // cs install binaries under .app-name.aux and scala-cli should drop .aux from progName
+          val binary       = "scala".prepareBinary(root)
+          val csBinaryName = root / ".scala.aux"
+          os.move(binary, csBinaryName)
+          val output   = os.proc(csBinaryName, "test", "--usage").call(check = false).out.text()
+          val usageMsg = TestUtil.removeAnsiColors(output)
+          expect(usageMsg.contains(" scala "))
```

Comment:
Maybe this assertion is too weak? Should it be something like `expect(!(usageMsg.contains(".scala.aux")))`

Reply:
Right, Thanks for review!